input/fsh/au-erequesting-diagnosticrequest.fsh

```diff
@@ -10,6 +10,20 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
 
 * ^extension[http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm].valueInteger = 0
 
+* extension contains http://hl7.org/fhir/StructureDefinition/patient-sexParameterForClinicalUse named sexParameterForClinicalUse 0..1 MS
```

Comment:
Commenting here to get an inline comment. 

Please update Terminology page to add value set/ code system resources referenced by the extension (the page lists the value sets bound as [preferred](https://hl7.org/fhir/R4/terminologies.html#preferred) or greater to a supported element, element slice, or extension, and the associated code systems.

Reply:
Done

Reply:
Change not included - may be a missing commit?

Reply:
Resolving this comment as specific feedback has been provided against the changes.